README.md

```diff
@@ -1,37 +1,53 @@
 ## Coq LSP
 
-The `coq-lsp` project aims to provide a lightweight, pure [Language Server
-Protocol](https://microsoft.github.io/language-server-protocol/)
-server implementation for the Coq proof assistant, as well as to serve
-as a framework for interface experimentation.
+`coq-lsp` is a new [Language
+Server](https://microsoft.github.io/language-server-protocol/) and [Visual
+Studio Code](https://code.visualstudio.com/) extension for the [Coq Proof
+Assistant](https://coq.inria.fr).
 
-**Warning**: This project is at an _early_ stage, and it has **known bugs**,
-see the issue tracker for more information.
+Key [features](#Features) of `coq-lsp` are continuous and incremental document
+checking, advanced error recovery, markdown support, positional goals and
+information panel, performance data, ...
 
-Contributions are very welcome, but please **first coordinate** with the dev
-team in Zulip before writing any code, as the code evolves pretty quickly.
+`coq-lsp` aims to provide a seamless, modern interactive theorem proving
+experience, as well as to serve as a platform for research and UI integration
+with other projects.
 
-## Development Channel
+### Installation
 
-Our development channel can be found at [Coq's
-Zulip](https://coq.zulipchat.com/#narrow/stream/329642-coq-lsp), don't hesitate
-to stop by.
+In order to use `coq-lsp` you'll need to install [**both**](etc/FAQ.md) the
+`coq-lsp` server, and the Visual Studio Code extension for it to recognize Coq
```

Comment:
```suggestion
`coq-lsp` server, and a client such as the Visual Studio Code extension for it to recognize Coq
```

Reply:
Thanks. I'm indeed struggling between the views to focus on `coq-lsp` being a VS Code client, or thinking about other clients. This is sublte as users don't understand this client/server stuff.

I'm leaning towards removing any mention of EMacs and other clients, until they really exist. While `coq-lsp` works on LSP-compliant clients, it is not usable without the goals stuff, which can be on the other hand implemented quite cheaply in most clients I'd dare to say.

So maybe indeed I'm gonna take that simpler view that (for now) `coq-lsp = server + VSCode client`